doc/docs.md

```diff
@@ -3635,7 +3635,8 @@ To improve safety and maintainability, operator overloading is limited:
 - `==` and `!=` are self generated by the compiler but can be overriden.
 - Calling other functions inside operator functions is not allowed.
 - Operator functions can't modify their arguments.
-- When using `<`, `>`, `>=`, `<=`, `==` and `!=` operators, the return type must be `bool`.
+- When using `<`, `<=` and `==` operators, the return type must be `bool`.
+- `!=`, `>` and `>=` are auto generated when `!=`, `<=` and `<` are defined.
```

Comment:
Small typo here, I suppose it should be `[...] auto generated when `==`, `<=` [...]`

Reply:
Yes 🙂

I will fix it tomorrow! But anyone can do this change right now then feel free to do so!

Reply:
Fixee